packages/embed/src/utils/set-auto-close.ts

```diff
@@ -0,0 +1,10 @@
+import { getFormSubmitHandler } from './create-iframe/get-form-event-handler'
+
+export const setAutoClose = (embedId: string, options: { autoClose?: number }, callback: () => void) => {
+  if (options.autoClose) {
+    window.addEventListener(
+      'message',
+      getFormSubmitHandler(embedId, () => setTimeout(callback, options.autoClose))
```

Comment:
What happens when the `autoClose` has no value, eg `<div data-tf-popup="id" data-tf-auto-close></div>`? Will it close? What will be the default timeout in this case? I think it should be zero and close right away.